tests/unit/graph_reader/expected/airflow_pipeline_a0_b0_dependencies/airflow_pipeline_a0_b0_dependencies_requirements.txt

```diff
@@ -0,0 +1,2 @@
+lineapy==0.1.4
+lineapy==0.1.4
```

Comment:
why are there duplicates here? i think we ensure that we dont have duplicate ones. also for lineapy the plugins dont include the exact version for lineapy (the tests break every time we try to release because the versions wont match)

Reply:
Looks like it is concatenating requirements from each session. In this case, it is from two sessions, so it is duplicated. I've fixed the problem. 

However, it actually raises an interesting question. What should we do if the requirements for two sessions conflict?